docs/builds.md

```diff
@@ -0,0 +1,77 @@
+# GitHub Actions - Builds
+
+GitHub for Jira supports builds via [GitHub Actions workflow syntax](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions).
+To set this up, you will need to create a .github folder at the root of a given repository and then make a child directory
+called workflows. Inside of .github/workflows you will need to create a build.yml file. This is where you will specify the workflow for your builds.
+
+Following is an example of a build.yml file:
+
+```
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+      - feature/**
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a single command using the runners shell
+      - name: Run a one-line script
+        run: echo Hello, world!
+
+      # Runs a set of commands using the runners shell
+      - name: Run a multi-line script
+        run: |
+          echo Add other actions to build,
+          echo test, and deploy your project.
+          sleep 60s
+```
+
+Once you have a similar file in any of your repositories that are connected to our app, you will start to see builds data
+in the development panel in Jira.
+
+![Builds data in Jira](./images/builds-data-jira-dev-panel.png)
+
+#### Supporting Multiple Commits with Issue Keys
+
+One important thing to note in the above example is the branches being targeted under `pull_requests` `branches`:
+
+```
+pull_request:
+    branches:
+      - main
+      - feature/**
+```
+
+In order for the GitHub for Jira app to be able to compare two points in your git history (possible once a pull request is opened)
```

Comment:
`In order for the Github for Jira app to compare` is a bit easier to read imo - nit

I would also remove the brackets and turn them into a new sentence to say 
``` 
In order for the GitHub for Jira app to be able to compare two points in your git history, we need a base branch and a head branch. This is only possible once a pull request is opened. When the app has both branches...
```
I'm not sure if this is what you were trying to say in the brackets but I felt it was unclear

Reply:
<img width="1321" alt="Screen Shot 2021-12-07 at 12 37 26 pm" src="https://user-images.githubusercontent.com/37155488/144950046-02e82552-e2ef-4b22-831d-d77208a8b0c1.png">
 
Just ran the suggestion through a grammar check and it's highlights that the additional words make the sentence too wordy. Will leave this as is.

Reply:
Scratch that. Ended up rewording that first part to make the flow between sentences make more sense.